packages/bits/src/lib/repeat/repeat-item/repeat-item.component.ts

```diff
@@ -23,7 +23,5 @@ export class RepeatItemComponent {
 
     @Input() public selected = false;
 
-    @Input() public draggable = false;
-
-    get role(): string { return this.clickable || this.draggable ? "option" : "listitem"; }
+    get role(): string { return this.clickable ? "option" : "listitem"; }
```

Comment:
I'm still not sure that `clickable` provides enough information to know whether the repeat item is a selectable `option` because it could be clickable in the sense that you can click it to navigate somewhere or do some other action besides selecting the item. The correct role may need to be an input whose value comes from the repeat component.

Reply:
Discussed offline and added `selectable` input for that purpose